src/app/campaign/flight/flight.container.ts

```diff
@@ -81,8 +81,20 @@ export class FlightContainerComponent implements OnInit, OnDestroy {
   }
 
   flightUpdateFromForm({ flight, changed, valid }) {
+    this.flightState$.pipe(
```

Comment:
filter to only load the availability when `flight` fields from form have changed as compared to the `localFlight` (different since availability was last loaded)
...maybe belongs in `campaignStore.setFlight()` instead?